package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "description": "A datepicker element based on Google's Material Design built from scratch with Polymer",
+  "keywords": [
+    "web-component",
+    "web-components",
+    "polymer",
+    "seed",
+    "calendar",
+    "date",
+    "picker",
+    "cal",
+    "date-picker",
+    "datepicker",
+    "motss"
+  ],
+  "homepage": "https://github.com/motss/app-datepicker/",
+  "name": "app-datepicker",
+  "version": "3.0.0",
+  "resolutions": {
+    "inherits": "2.0.3",
+    "samsam": "1.1.3",
+    "supports-color": "3.1.2",
+    "type-detect": "1.0.0"
+  },
+  "main": "app-datepicker.js",
+  "author": "motss <wes_ngrongsen@hotmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "@polymer/iron-iconset-svg": "^3.0.0-pre.18",
+    "@polymer/iron-list": "^3.0.0-pre.18",
+    "@polymer/iron-selector": "^3.0.0-pre.18",
+    "@polymer/neon-animation": "^3.0.0-pre.18",
+    "@polymer/paper-dialog-behavior": "^3.0.0-pre.18",
+    "@polymer/paper-icon-button": "^3.0.0-pre.18",
+    "@polymer/polymer": "^3.0.0",
+    "web-animations-js": "^2.3.1",
+    "@polymer/paper-button": "^3.0.0-pre.18"
```

Comment:
All Polymer 3 based components should be out of beta. Please upgrade those to the latest.